PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -3509,7 +3514,16 @@ struct HfTrackIndexSkimCreatorLfCascades {
           auto trackParVarPion1 = getTrackParCov(trackPion1);
 
           // find charm baryon decay using xi PID hypothesis
-          int nVtxFrom2ProngFitterXiHyp = df2.process(trackCascXi2Prong, trackParVarPion1);
+          int nVtxFrom2ProngFitterXiHyp = 0;
+          try {
+            nVtxFrom2ProngFitterXiHyp = df2.process(trackCascXi2Prong, trackParVarPion1);
+          } catch (...) {
+            LOGF(info, "Exception caught: failed to find charm baryon vertex (2prong - xi)");
+            registry.fill(HIST("hFitterStatusXi2Prong"), 1);
```

Comment:
Shouldn't you check `if (fillHistograms)` before? Otherwise the histogram is not added to the registry and it should crash.

Reply:
Yes, thanks, I overlooked it :)